app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -79,6 +80,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 */
 		var resumeSubtractDuration = stringPreference("pref_resume_preroll", "0")
 
+		/**
+		 * Foreground Color for the Subtitles
+		 */
+		var subtitlesTextColor = longPreference("subtitles_text_color", 0xFFFFFFFF)
+
```

Comment:
Probably makes sense to move this to where the other subtitle related preferences are